src/error/error.rs

```diff
@@ -75,38 +87,41 @@ impl ErrorKind {
         ErrorKind { kind: BadRequest }
     }
 
-    /// Returns `true` if `self` represents a 400 -- bad request error
-    pub fn is_bad_request(&self) -> bool {
```

Comment:
These functions need to stay as they are part of the public API. They can be considered for removal in the next breaking release.

Reply:
Ok, I see.